service/matching/config.go

```diff
@@ -69,6 +69,10 @@ type (
 		EnableTaskInfoLogByDomainID dynamicconfig.BoolPropertyFnWithDomainIDFilter
 
 		ActivityTaskSyncMatchWaitTime dynamicconfig.DurationPropertyFnWithDomainFilter
+
+		// isolation configuration
+		EnableTasklistIsolation dynamicconfig.BoolPropertyFnWithDomainFilter
+		AllIsolationGroups      []string
```

Comment:
Nonblocking, but since we're using this on startup in two libraries and these matching changes I'd be leaning to just putting this as part of the shared service resource config and loading it at startup so there's a single source of truth.